### PR TITLE
Var assign

### PR DESCRIPTION
**Pull request by lucianmot**

### CODE REVIEW

**Review by aniasobo — APPROVED** [review]
OMG IT WORK

**Review by zepherine2006DEV — APPROVED** [review]
You've only gone and done it!! Yay!!!!!

File: interpreter_test.py [hunk 1]
```diff
@@ -250,7 +251,13 @@ def test_tokeniser_tokenises_fizzbuzz_statement_with_end(self):
         from forest import Tokeniser
         tokeniser = Tokeniser("WALK_PATH_IF_SEE^30^(*)>^15^OvO^0^echo^<<fizzbuzz>>^CAMP")
         self.assertEqual(tokeniser.create_tokens(), [{"IF_START" : "WALK_PATH_IF_SEE"}, {"INTEGER" : "30"}, {"MODULUS" : "(*)>"}, {"INTEGER" : "15"}, {"EQUALS" : "OvO"}, {"INTEGER" : "0"}, {"ECHO" : "echo"}, {"STRSTART" : "<<"}, {"STRING_CONTENT" : "fizzbuzz"}, {"STRSTOP" : ">>"}, {"END" : "CAMP"}])
-        
+
+    def test_tokenise_variable_assigment(self):
+        from forest import Tokeniser
+        tokeniser = Tokeniser("BACKPACK")
```
Comment (josephtownshend) [inline interpreter_test.py:304]: super cool

**Review by josephtownshend — COMMENTED** [review]
wickedz

**Pull request merged**